sw/device/examples/hello_usbdev/hello_usbdev.c

```diff
@@ -82,6 +83,8 @@ static dif_gpio_t gpio;
 
 int main(int argc, char **argv) {
   uart_init(kUartBaudrate);
+  base_set_stdout(uart_stdout);
```

Comment:
Can this be placed inside `base_log_internal_core()` instead?

Reply:
No. The logging functions are output agnostic, so it's the binary's responsibility to set up a stdout. By default, logging does nothing, after all.